app/views/providers/uploaded_evidence_collections/_uploaded_files.html.erb

```diff
@@ -23,8 +20,6 @@
     </thead>
     <tbody class="govuk-table__body">
 
-    <span alert-live="assertive" class="govuk-visually-hidden"><%= success_message %></span>
```

Comment:
ARIA live regions are generally used for accessibility purposes, are you sure this no longer required?
If required, it is best to move it to elsewhere in DOM as this isn't a valid location.